packages/ors/index.ts

```diff
@@ -134,6 +142,15 @@ export class ORS implements BaseRouter {
                     return res
                 }
             })
+            .catch((error) => {
+                const props: ORSErrorProps = {
```

Comment:
Check the docs to see if the error response is the same across endpoints, so you could define the callback once and just pass it to `.catch()` each time.